hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1235,7 +1235,7 @@
       uvm_test_seq: "chip_sw_pwrmgr_deep_sleep_all_wake_ups_vseq"
       sw_images: ["//sw/device/tests/sim_dv:pwrmgr_normal_sleep_all_wake_ups:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+sw_test_timeout_ns=18_000_000"]
+//      run_opts: ["+sw_test_timeout_ns=18_000_000"]
```

Comment:
if you don't need it, can you remove it?

Reply:
Done